specification.md

```diff
@@ -2543,12 +2571,26 @@ workflowId: handleApprovedVisaWorkflowID
 
 </details>
 
+The `workflowId` property defined the unique ID of the sub-workflow to be invoked.
+
+The `version` property defined the unique version of the sub-workflow to be invoked.
+If this property is defined, runtimes should match both the `id` and the `version` properties
+defined in the sub-workflow definition. 
+
+The default value of `version` is `"latest"`, meaning that if this property is not defined
+the lookup should be done by matching the specified `workflowId` and the latest version 
+of the sub-workflow. Runtimes are responsible to provide this "latest" version in case of 
+multiple sub-workflow definitions matching the `workflowId`. In the cases where no 
+sub-workflow can be found that match the defined criteria, or if there is ambiguity in the cases
+where multiple sub-workflow definitions match the criteria, the runtimes should report this as 
+an error.
+
 The `waitForCompletion` property defines if the SubFlow action should wait until the referenced reusable workflow
 has completed its execution. If it's set to "true" (default value), SubFlow action execution must wait until the referenced workflow has completed its execution.
 In this case the workflow data output of the referenced workflow will be used as the result data of the action.
-If it is set to "false" the parent workflow can continue its execution as soon as the referenced sub-workflow 
-has been invoked (fire-and-forget). For this case, the referenced (child) workflow data output will be ignored and the result data 
-of the action will be an empty json object (`{}`).
+In the case it's set to "false" the SubFlow action can finish its execution as soon as the referenced sub-workflow 
+has been invoked (fire-and-forget). For this case, the referenced sub-workflow data output should be ignored and the result data 
```

Comment:
> output should be ignored

Nit; in RFC's they're very careful about the words 'should' vs. 'must'.  I think we must avoid the word 'should' (in general, but here as a particular example) as it could be construed as something that is a suggestion and not a requirement of the spec.

Reply:
Or rather, I think we should be careful about the use of words like "should" vs. "must".  In this case, this seems like a "must" rather than a "should".  WDYT?

Reply:
there is https://tools.ietf.org/html/rfc2119
we are planning to update our documentation to use it